src/Voron/Data/CompactTrees/CompactTree.Debug.cs

```diff
@@ -132,7 +132,7 @@ private void VerifyNode(ref CursorState current)
 
         for (int i = 1; i < current.Header->NumberOfEntries; i++)
         {
-            GetEncodedEntry(current.Page, current.EntriesOffsets[i], out var encodedKey, out var encodeKeyLengthInBits, out l);
+            var encodedKey = GetEncodedKeySpan(ref current, i, out var encodeKeyLengthInBits, out l);
```

Comment:
Same warning here.

Reply:
We cannot mix span allocated via the  heap  and the stack :-(

Fixed